src/main/java/com/conveyal/datatools/manager/auth/Auth0Users.java

```diff
@@ -48,20 +49,20 @@
     /**
      * Overload of user search query URL to restrict search to current users only.
      */
-    private static URI getUrl(String searchQuery, int page, int perPage, boolean includeTotals) {
-        return getUrl(searchQuery, page, perPage, includeTotals, true);
+    private static URI getSearchUrl(String searchQuery, int page, int perPage, boolean includeTotals) {
+        return getSearchUrl(searchQuery, page, perPage, includeTotals, true);
     }
 
     /**
      * Constructs a user search query URL.
      * @param searchQuery   search query to perform (null value implies default query)
      * @param page          which page of users to return
-     * @param perPage       number of users to return per page
+     * @param perPage       number of users to return per page (limited to 1000)
      * @param includeTotals whether to include the total number of users in search results
      * @param limitToCurrentUsers whether to restrict the search to current users only
      * @return              URI to perform the search query
      */
-    private static URI getUrl(String searchQuery, int page, int perPage, boolean includeTotals, boolean limitToCurrentUsers) {
+    private static URI getSearchUrl(String searchQuery, int page, int perPage, boolean includeTotals, boolean limitToCurrentUsers) {
```

Comment:
Suggestion: rename `perPage` to `itemsPerPage` or something similar.

Reply:
I think I'd prefer to keep it how it is so that it matches the param name in the Auth0 docs/endpoint.